<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoView</class>
 <widget class="QWidget" name="VideoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Video">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>460</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
	border : 1px solid black;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Seek_Backward">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border : none;
	border-radius : 15px;
	min-width : 60px;
	max-width : 60px;
	min-height : 60px;
	max-height: 60px;
	background-color : rgb(255,255,255);
	color : rgb(52,51,64);
	padding : 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Play_Pause">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border : none;
	border-radius : 15px;
	min-width : 60px;
	max-width : 60px;
	min-height : 60px;
	max-height: 60px;
	background-color : rgb(255,255,255);
	color : rgb(52,51,64);
	padding : 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Stop">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>570</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border : none;
	border-radius : 15px;
	min-width : 60px;
	max-width : 60px;
	min-height : 60px;
	max-height: 60px;
	background-color : rgb(255,255,255);
	color : rgb(52,51,64);
	padding : 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Seek_Forward">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>570</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border : none;
	border-radius : 15px;
	min-width : 60px;
	max-width : 60px;
	min-height : 60px;
	max-height: 60px;
	background-color : rgb(255,255,255);
	color : rgb(52,51,64);
	padding : 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Volume">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>570</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border : none;
	border-radius : 15px;
	min-width : 60px;
	max-width : 60px;
	min-height : 60px;
	max-height: 60px;
	background-color : rgb(255,255,255);
	color : rgb(52,51,64);
	padding : 0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_Volume">
   <property name="geometry">
    <rect>
     <x>629</x>
     <y>594</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal
{
	height : 5px;
	width : 140px;
	background : red;
	border-radius : 2px;
}
QSlider::handle:horizontal
{
	background : rgb(255,255,255);
	width : 10px;
	height : 20px;
	margin : -5px -5px;
	border-radius : 5px;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_Duration">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal
{
	height : 10px;
	width : 550px;
	background : red;
	border-radius : 5px;
}
QSlider::handle:horizontal
{
	background : rgb(255,255,255);
	width : 20px;
	height : 40px;
	margin : -5px -5px;
	border-radius : 10px;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Current_Time">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Total_Time">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>540</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_File_Name">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>811</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color : rgb(255,255,255);
	border : 2px solid rgb(255,255,255);
}</string>
   </property>
   <property name="text">
    <string>File Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
